compiler/rustc_borrowck/src/diagnostics/mod.rs

```diff
@@ -1,8 +1,10 @@
+// #![deny(rustc::untranslatable_diagnostic)]
+// #![deny(rustc::diagnostic_outside_of_impl)]
```

Comment:
If we're going to land this without the entire crate being completed, then it'd would be good to have this comment removed.